tests/standalone_apps/discovery_test_plugin/setup.py

```diff
@@ -8,5 +8,5 @@
     author_email="omry@fb.com",
     url="https://github.com/facebookresearch/hydra/",
     packages=find_namespace_packages(include=["hydra_plugins.*"]),
-    install_requires=["hydra-core==1.0.*"],
+    install_requires=["hydra-core==1.*"],
```

Comment:
```suggestion
    install_requires=["hydra-core"],
```

Reply:
Resolved in https://github.com/facebookresearch/hydra/pull/1197/commits/42f059f494206bf1f82cd3892c0a1832581655ef